### DISCUSSION TITLE
Hide Mute tab button

### DISCUSSION THREAD

**Discussion by Minnona** [piroor/treestyletab#3278] [Q&A] [answered]
Hey, I'm trying to make a small sidebar, and, so far, it's been great except for one thing - playing videos on YouTube or any other website with media playing would show a mute button instead of favicon. How can I disable this button completely? 
![image](https://user-images.githubusercontent.com/23418072/211370030-ed027afc-81c3-49c3-9b74-644eb51a5dae.png)

**Comment by piroor** [comment]
It depends on what user styles you currently use (TST's advanced option and userChrome.css). Please post them here for more effective answer.

**Comment by Minnona** [comment]
[tts.txt](https://github.com/piroor/treestyletab/files/10388705/tts.txt)
[userChrome.txt](https://github.com/piroor/treestyletab/files/10388708/userChrome.txt)

**Comment by irvinm** [comment, marked as answer]
There is obviously a ton of CSS you are using, but in general if you want to hide the sound button this will work.

```
.sound-button:after,
.sound-button {
    display: none !important;
}
```

If you search your TST CSS, there are 43 instances of "sound-button" ... so obviously there is a lot of customization going on.  Maybe start by placing the block at the very end.  I did a basic check by doing that and the button no longer was shown.

**Reply by Minnona** [reply]
It worked, thank you!
